components/common/src/main/java/com/hotels/styx/common/logging/HttpRequestMessageLogger.java

```diff
@@ -36,11 +35,23 @@ public HttpRequestMessageLogger(String name, boolean longFormatEnabled) {
     }
 
     public void logRequest(HttpRequest request, Origin origin) {
-        logger.info("requestId={}, request={}", request.id(), information(request, origin, longFormatEnabled));
+        if (request == null) {
+            logger.warn("requestId=N/A, request=null, origin={}", origin);
+        } else {
+            logger.info("requestId={}, request={}", request.id(), information(request, origin, longFormatEnabled));
+        }
     }
 
     public void logResponse(HttpRequest request, HttpResponse response) {
-            logger.info("requestId={}, response={}", request.id(), information(response, longFormatEnabled));
+        if (response == null) {
+            logger.warn("requestId={}, response=null", id(request));
```

Comment:
Hi @kvosper the warning on line 39 uses "N/A" in place of "null", but the warning on line 47 uses "null".
Let's be consistent with this and either use "N/A" or "null" everywhere, but let's not mix and match.